docs/linking-to-pfam.rst

```diff
@@ -1,58 +1,22 @@
-.. _linking-to-pfam:
-
 ***************************
 Linking to the Pfam website
 ***************************
 
-Linking to family pages
+Linking to Pfam entry pages
 =======================
 
-You can refer to Pfam families either by accession or ID. You can also refer to a family by "entry", although this is a convenience that should be used only if you're not sure if what you have is an accession or an ID.
-
-Pfam accession numbers are more stable between releases than IDs and we strongly recommend that you link by accession number.
-
-Here are some examples of linking to Pfam families:
-
-Directly, using either accession or ID:
-    `/family/PF00002 <http://pfam.xfam.org/family/PF00002>`_ or
-    `/family/7tm_2 <http://pfam.xfam.org/family/7tm_2>`_ 
-Using a parameter, by accession:
-    `/family?acc=PF00002 <http://pfam.xfam.org/family?acc=PF00002>`_ 
-Using a parameter, by ID:
-    `/family?id=7tm_2 <http://pfam.xfam.org/family?id=7tm_2>`_ 
-Using the "entry" parameter with either accession or ID:
-    `/family?entry=PF00002 <http://pfam.xfam.org/family?entry=PF00002>`_ or
-    `/family?entry=7tm_2 <http://pfam.xfam.org/family?entry=7tm_2>`_ 
+You can refer to Pfam entries using their accession. For example `/entry/pfam/PF00002 <https://www.ebi.ac.uk/interpro/entry/pfam/PF00002/>`_
```

Comment:
maybe it's more clear adding the InterPro page here? (as before the accession...)

Reply:
what do you like about this page?

Reply:
for example

```
Example for atlas pages

http://www.ebi.ac.uk/pdbe-srv/view/entry/????/summary.html
http://www.ebi.ac.uk/pdbe-srv/view/entry/1mbd/summary.html 
```

this way of showing examples with the whole url, personally find it more clear but guess it's personal just a suggestion :)

Reply:
done, let me know if that's better

Reply:
Yes I find it all more clear now, I like it :)